src/java/voldemort/store/readonly/ReadOnlyStorageEngine.java

```diff
@@ -72,6 +73,7 @@
     private volatile ChunkedFileSet fileSet;
     private volatile boolean isOpen;
     private long lastSwapped;
+    private AtomicBoolean currently_fetching;
```

Comment:
Let's stick with camel case (i.e.: camelCase, not underscore_separation) so that we're more inline with Java coding style conventions.